cosmoz-omnitable-treenode-column.html

```diff
@@ -112,6 +120,36 @@
 			_tooltip: {
 				type: String,
 				computed: '_computeTooltip(filter)'
+			},
+
+			/**
+			 * `<paper-autocomplete>` `queryFn`
+			 * Seach for indexOf query in all the text
+			 */
+			queryFn: {
+				type: Function,
+				value: function () {
```

Comment:
To go all-in ES6, this would be:

```js
value: () =>
  (datasource, query) =>
    datasource.map(item => {
      ...
    })
    .filter(item => item !== undefined)
```
Maybe a bit hard to read though? What do you think?

Either way we should use const/let instead of var.

Reply:
Perfect I will apply and I will try to get used with () =>  . I had an intention to use let instead of var but I wasn't sure if it is backward compatible.